src/components/Profile/Profile.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../assets/MainStyle.scss';
+
+.main {
+  width: 70%;
+  padding-left: 2%;
+  font-family: 'Open Sans', sans-serif;
+}
+  
+.profile-tab-menu {
+  .menu {
```

Comment:
Nice css here @nadralia. However, according to the conventions, we are using [BEM](https://github.com/andela/ah-frontend-dojo/wiki/BEM).

so this line could be `.profile-tab-menu {
&__menu {`